test/org/sagebionetworks/bridge/play/controllers/StudyControllerTest.java

```diff
@@ -345,6 +371,30 @@ public void getSummaryStudiesWithSummaryWorks() throws Exception {
         // Throw an exception if the code makes it this far.
         doThrow(new RuntimeException()).when(controller).getAuthenticatedSession(ADMIN);
     }
+
+    @Test
+    public void getSummaryStudiesWithInactiveOnes() throws Exception {
```

Comment:
We need to validate in this test that we never try to getAuthenticatedSession(ADMIN).